zilliqa/src/time.rs

```diff
@@ -58,6 +59,12 @@ mod time_impl {
         }
     }
 
+    impl From<DateTime> for SystemTime {
```

Comment:
Don't you need this when `#[cfg(not(feature = "fake_time"))]` too?

Reply:
The `impl From<DateTime> for SystemTime {` implementation already exists in `datetime`.

Confirmed with build with `--build-arg="is_release=true"`, that's correct right?

Reply:
Ah sorry I missed that